commands/operator-sdk/cmd/test/cluster.go

```diff
@@ -62,7 +62,7 @@ func testClusterFunc(cmd *cobra.Command, args []string) error {
 	// in main.go, we catch and print errors, so we don't want cobra to print the error itself
 	cmd.SilenceErrors = true
 	if len(args) != 1 {
-		return fmt.Errorf("operator-sdk test cluster requires exactly 1 argument")
+		log.Fatalf("Command %s requires exactly one argument", cmd.CommandPath())
```

Comment:
I'll switch this back.

Reply:
On second thought, I'm going to make all errors log statements in this function and have the function not return an `error`, since we `log.Fatal` errors in all other CLI functions except this one.

Reply:
> On second thought, I'm going to make all errors log statements in this function and have the function not return an `error`, since we `log.Fatal` errors in all other CLI functions except this one.

Using `return fmt.Errorf` would be more correct and cleaner though. It would allow use to use cobra's error output and usage statements. For example, if a user doesn't correctly specify the argument for `test cluster`, the output is:
```console
$ operator-sdk test cluster
Usage:
  operator-sdk test cluster <image name> [flags]

Flags:
  -h, --help                       help for cluster
      --image-pull-policy string   Set test pod image pull policy. Allowed values: Always, Never (default "Always")
      --kubeconfig string          Kubeconfig path
      --namespace string           Namespace to run tests in
      --pending-timeout int        Timeout in seconds for testing pod to stay in pending state (default 60s) (default 60)
      --service-account string     Service account to run tests on (default "default")

FATA[0000] operator-sdk test cluster requires exactly 1 argument
```

instead of:

```console
$ operator-sdk test cluster
Command test cluster requires exactly one argument
```

 I would prefer keeping this function returning an error and making the other commands return errors as well.

Reply:
That works for me. I'll revert this change and make a separate PR for converting logs to returned errors.